ruby/files_and_serialization/project_event_manager.md

```diff
@@ -1134,8 +1134,8 @@ For each of our attendees we want to replace the `FIRST_NAME` and `LEGISLATORS`
 * We need to find all instances of `LEGISLATORS` and replace them with the individual's representatives.
 
 Our template is a String of text which has two methods for replacing text:
-[String#gsub](http://rubydoc.info/stdlib/core/String#gsub-instance_method) and
-[String#gsub!](http://rubydoc.info/stdlib/core/String#gsub%21-instance_method).
+[String#gsub](http://rubydoc.info/stdlib/core-3.1.2/String#gsub-instance_method) and
```

Comment:
A few more in this file, I'll stop here pointing out each one individually :laughing: